docs/core/services/upload.service.md

```diff
@@ -119,3 +119,18 @@ It is also possible to provide the `versioningEnabled` value as part of the [`Fi
 
 > Note: When creating a new node using multipart/form-data by default versioning is enabled and set to MAJOR Version.
 > Since Alfresco 6.2.3 versioningEnabled flag was introduced offering better control over the new node Versioning.
+
+### Concurrent Uploads
+
+By default, the Upload Service processes one file at a time.
+You can increase the number of concurrent threads by changing the `upload.threads` configuration parameter:
```

Comment:
Are there any browser-specific considerations in terms of reasonable setting for this value ? Or should we mention testing in various scenarios (&/or browser), eg. if increasing beyond say 3, 4 or 5 ?

Reply:
I would say admins should experiment and decide on their own what is the appropriate number. it's too many variables here to suggest anything specific.

Reply:
Ok, for demo shell ADF.

I suggest we override for ADW, eg start at 2 or 3 once tested & sanity checked. See other comment.